lib/index.ts

```diff
@@ -4,7 +4,7 @@ function getAllSwears(): Array<swearVariants> {
     return [
         [`tabarnak`, `tabarnouche`, `tabarouette`, `taboire`, `tabarslaque`],
         [`câlisse`, `câlique`, `câline`, `câline de bine`, `câliboire`, `caltor`],
-        [`crisse`, `christie`, `crime`],
+        [`crisse`, `christie`, `crime`, `bout d'crisse`],
```

Comment:
`bout d'crisse' is actually already on line 31, but I agree that it should be here instead. Want to delete the other line and push again?

Reply:
@henrymyers done!